docs/reference/cli.md

```
@@ -875,6 +911,16 @@ codelens    default      attached    2h ago       1
 codelens    debug        detached    30m ago      2
 ```
 
+Remote Control (`rc-*`) sessions are enriched with metadata read from the in-shed
+`shed-ext-rc` binary, adding `KIND` and `RC-STATE` columns (and an `rc` object in
+`--json`). Enrichment degrades silently to the plain tmux row if a shed is unreachable
+or lacks the binary.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)